Functions.Templates/Templates-v2/EventHubTrigger-Python/blueprint_import.py

```diff
@@ -0,0 +1 @@
+from $(FILE_NAME) import $(FILE_NAME)
```

Comment:
There is no `$(FILE_NAME)` token defined in template.json - did you mean `$(BLUEPRINT_FILENAME)`?
```suggestion
from $(BLUEPRINT_FILENAME) import $(BLUEPRINT_FILENAME)
```